### DISCUSSION TITLE
Can't find xformers in the memory attention option，

### DISCUSSION THREAD

**Discussion by tangtaojim** [d8ahazard/sd_dreambooth_extension#1179] [Q&A]
### Already installed torch: 2.0.0+cu118 xformers: 0.0.17rc482，，Can't find xformers in the memory attention option，Can someone tell me why? Thanks

**Comment by ablasphemingheretic** [comment]
Do you have any other errors or dependency issues mentioned when you start the Web UI?  Is Automatic 1111 up-to-date?  I had this same issue but it's there now after updating the torch and installing xformers 0.0.18.  I had also deleted the venv folder and had it re-created when starting the webui as part of resolving another issue, which could've had something to do with it.  

Also, I had xformers running after following a guide I found somewhere, and the webui log stated that it was running "Applying xformers cross attention optimization," but the "--xformers" argument was not in the webui.bat file, interestingly, which seems to be all that most people are doing to enable it (provided they have the requisite RTX graphics cards).

**Reply by tangtaojim** [reply]
Thanks，I also reinstalled venv folder，Now it is normal，Although I don't know what the problem is，
